ENIGMAsystem/SHELL/Platforms/SDL/Window.cpp

```diff
@@ -32,6 +31,19 @@ namespace enigma {
 void (*WindowResizedCallback)();
 
 SDL_Window* windowHandle = nullptr;
+unsigned sdl_window_flags = SDL_WINDOW_HIDDEN;
+
+void init_sdl_window_attributes();
```

Comment:
This would probably just warrant a comment about how this is graphics bridge data... maybe a namespace would accomplish the same.

Reply:
Here: a913201f9ea18660df032ba066c07b4de12d8ffe

An additional C namespace and a comment should do it.